### PR TITLE
fix(guest-calling): changes for guest call implementation

### PR DESCRIPTION
**Pull request by Kesari3008**
Updated changes to implement guest calling: Webex Click to call.

### CODE REVIEW

File: agent.html [hunk 1]
```diff
@@ -383,7 +383,7 @@ <h3>Completed</h3>
 		<audio id="agent-remote-audio" autoplay></audio>
 		<!-- //blog -->
     <script src="https://unpkg.com/webex/umd/webex.min.js"></script>
-	<script src='https://unpkg.com/webex@2.60.0-next.2/umd/calling.min.js'></script>
+	<script src='https://unpkg.com/webex@3.5.0-next.25/umd/calling.min.js'></script>
```
Comment (sreenara) [inline agent.html:386]: Please change this to 3.7.0.. that supports guest calling right?

File: index.html [hunk 2]
```diff
@@ -167,7 +167,7 @@ <h4>New Zealand</h4>
 	</div>
 	<script src="js/SmoothScroll.min.js"></script>
 	<script src="https://unpkg.com/webex/umd/webex.min.js"></script>
-	<script src='https://unpkg.com/webex@2.60.0-next.2/umd/calling.min.js'></script>
+	<script src='https://unpkg.com/webex@3.5.0-next.25/umd/calling.min.js'></script>
```
Comment (sreenara) [inline index.html:170]: same here

File: js/app.js [hunk 3]
```diff
@@ -11,6 +11,10 @@ const secondCallTimer = document.querySelector('#secondCallNotification #call-ti
 const profileDropDown = document.getElementById("myDropdown");
 const profileOnline = document.querySelector(".dropbtn #availability");
 
+let service_app_token = 'MzA3MWQ4NGEtNDA5Ni00MDIwLThkOTctOGZlMmI1N2M1ODk5ZTA0NjAyZDYtNTk5_PF84_ef70fae6-b079-45e6-901e-0a1ba9856721'; // 6100 queue
+// let service_app_token = 'YTgyZjkxNTMtM2IzOC00NjYwLTlhMmQtMjk2Y2EzMjUyY2Y2MGFkN2ZkNjAtZTc5_P0A1_b657fcf3-6b06-46a4-ab0e-f23fceb7a447'; //5200 queue
+// const refresh_token = 'ZDQzZDE3MjgtYThmNS00NzkwLThhZGMtYjdmYzMxNjAyOTQwZmM1ZGY4NDgtOTU4_PF84_ef70fae6-b079-45e6-901e-0a1ba9856721';
```
Comment (sreenara) [inline js/app.js:16]: Please remove sensitive token info and only leave CONSTANTS which devs have to fill up. Similar to our instructions in the beta

File: js/app.js [hunk 4]
```diff
@@ -97,14 +101,92 @@ function openCallNotification(callObj) {
...
+//   };
+
+//   fetch("https://webexapis.com/v1/access_token", request)
+//     .then((response) => response.json())
+//     .then((result) => {
+//       console.log(result);
+//       service_app_token = result.access_token;
+//     })
+//     .catch((error) => console.error(error));
+// }
```
Comment (sreenara) [inline js/app.js:128]: Remove commented code please

File: js/app.js [hunk 5]
```diff
@@ -97,14 +101,92 @@ function openCallNotification(callObj) {
...
...X0F0UGhXeU1iVW01anA3OGVNYmdWX2ZPQVZfSjRMbGpGUW5VMEpvbzNybGcxZXpkdG1kZld1SFZNTFpyOVk2TV92WHpjeDBoWE9TaF93Wms4Y0xEZmF5T3NjWHZpNWdHMUxfLU52bXdYRnVyZEFCZHJFRm1WeUx2N3c3OV9xODczb0RkeDNsSndLWS43YzU2MnV2SDQ5LWRhRDRwcTQ5OXRRIiwidXNlcl90eXBlIjoidXNlciIsInRva2VuX2lkIjoiQWFaM3IwWWpJMVl6ZGhZakF0TnpjNU15MDBZelE1TFdKaU1EWXRPVE16T1RWalpUWmlZalExTURJelpEQmtOMlF0TURFeSIsInJlZmVyZW5jZV9pZCI6ImFjY2EyMTUyLThmYjAtNDBmYy05NjQzLTk1NDIyODk5YjEyZiIsImlzcyI6Imh0dHBzOlwvXC9pZGJyb2tlci1iLXVzLndlYmV4LmNvbVwvaWRiIiwidXNlcl9tb2RpZnlfdGltZXN0YW1wIjoiMjAyMzA2MDkxMDAyNDQuMzgyWiIsInJlYWxtIjoiYjY1N2ZjZjMtNmIwNi00NmE0LWFiMGUtZjIzZmNlYjdhNDQ3IiwiY2lzX3V1aWQiOiIwM2NmOWQzNC02ZGE5LTQxNTctYmRlZS1iODg0NzQ3Zjk1Y2IiLCJ0b2tlbl90eXBlIjoiQmVhcmVyIiwiZXhwaXJ5X3RpbWUiOjE3Mjg0MzcyNTI3MjcsImNsaWVudF9pZCI6IkM2NGFiMDQ2MzllZWZlZTQ3OThmNThlN2JjM2ZlMDFkNDcxNjFiZTBkOTdmZjBkMzFlMDQwYTZmZmU2NmQ3ZjBhIn0.TPyBpG7IAhfCrU-QWzPJJi7dWQllvCFhpXFl8YAscuqnbHVdSs-L0AXgDR9WbtOVI1lq7NWS2suJGFIhYrEpAqa90pftx56T5ldK9cnL6TxAVmi9re-S3Eu-2cBAdUVa8GAj84LkLO_3DKKkuijqiRur0uY39ymt9L5Ljay_hAveKJF3MV_YX15b_LKGGCoFgaDBCi8DHUCQiFXQEezB6bNDrTJPH7yteBZvFdEh39mH2XsJ7fK9cJqkuDVCfysSq0aSD_PlyniU2Jf2rYhov5XPWqQjhDBfm3BQksTC7rfOTYBYTnUINOS1H-YRRlT9haleMHkZAKpmKOMyTHn6KQ";
```
Comment (sreenara) [inline js/app.js:183]: Remove all sensitive info

**Review by sreenara — CHANGES_REQUESTED** [review]

File: js/app.js (hunk 5 above)
Comment (Kesari3008) [reply js/app.js]: done

File: js/app.js (hunk 3 above)
Comment (Kesari3008) [reply js/app.js]: done

File: agent.html (hunk 1 above)
Comment (Kesari3008) [reply agent.html:386]: Even 3.5.0 supports guest calling

**Review by sreenara — APPROVED** [review]

File: js/app.js (hunk 4 above)
Comment (Kesari3008) [reply js/app.js]: removede

**Pull request merged**